packages/flutter_tools/lib/src/asset.dart

```diff
@@ -406,11 +689,11 @@ class _Asset {
   /// A platform-independent URL representing the entry for the asset manifest.
   final Uri entryUri;
 
-  File get assetFile {
-    return globals.fs.file(globals.fs.path.join(baseDir, globals.fs.path.fromUri(relativeUri)));
+  File createAssetFile(FileSystem fileSystem) {
```

Comment:
This name sounds like it's creating the file.

Reply:
changed to `lookupAssetFile` which hopefully sounds more like what it does